src/lib/secrets/keypair_common.ml

```diff
@@ -34,25 +32,15 @@ struct
   let read_exn ?(should_reask = true) ~which path =
     let read_privkey password = read ~privkey_path:path ~password in
     let%bind result =
-      match (Sys.getenv env, Sys.getenv env_deprecated) with
-      | Some password, _ ->
+      match Sys.getenv env with
+      | Some password ->
           (* this function is only called from client commands that can prompt for
              a password, so printing a message, rather than a formatted log, is OK
           *)
           printf "Using %s private-key password from environment variable %s\n"
             which env ;
           read_privkey (lazy (Deferred.return @@ Bytes.of_string password))
-      | None, Some password ->
-          (* this function is only called from client commands that can prompt for
-             a password, so printing a message, rather than a formatted log, is OK
-          *)
-          printf
-            "Using %s private-key password from deprecated environment \
```

Comment:
I have the same concern here. Shouldn't we log an error saying this variable is no longer supported instead?

Reply:
Same comment here: this allows the deprecated variable to be used, until such time as we remove it.